_integration/testsuite/run-test-case.sh

```diff
@@ -33,5 +33,5 @@ exec "$TESTER" run \
     --param proxy.address="$ADDRESS" \
     --param proxy.http_port="$HTTP_PORT" \
     --param proxy.https_port="$HTTPS_PORT" \
-    --watch pods,secrets \
+    --watch pods,secrets,configmaps \
```

Comment:
Looks like this might be a change when you were testing an integration test?

Reply:
Yep, though I added it to this PR on purpose. I can remove it.